plutarch-test/plutarch-base/Plutarch/ApiSpec.hs

```diff
@@ -47,6 +48,10 @@ spec = do
             $ \case
               PMinting c -> popaque c
               _ -> perror
+    describe "value" $ do
+      pgoldenSpec $ do
+        "singleton" @| PValue.singleton # pconstant "c0" # pconstant "sometoken" # 1 @-> \p ->
+          plift p @?= mint
```

Comment:
Can we add goldens for all the new functions in this PR, not just `singleton`?

Reply:
Of course, that's why this is a draft PR.

Reply:
Ah, yes, I hadn't seen the draft status.